### DISCUSSION TITLE
Modeling solidification of water vapor similar to water vapor condensation at colder environment -100 C to 0 C

### DISCUSSION THREAD

**Discussion by Muthu-V** [firemodels/fds#13852] [Q&A]
@drjfloyd ,  @mcgratta  Presently when we define water vapor species  as an aerosol , it is condensing and tracks  a species variable 'WATER VAPOR_COND '  . It is useful to check FOG formation and obscurity in colder atmosphere. Similarly,  Is it possible to track an aerosol species for solidification of water vapor resulting as ice particles.  The attached picture shows the formation of condensation near zero degrees. But at the very cold regions its appropriate to form ice Particles marked in the red square. Let me know if there is any possibility to model that part via aerosol model ( clean way ) by tracking solid species or injecting the solid particles in each cell if there is a H20 vapor and cell temperature below 0 C and subtract the latent heat of solidification of ice from the background carrier flow. The obscurity that is formed by ice particles is important in the near field of the jet. Any thoughts how to model in FDS  ?

![image](https://github.com/user-attachments/assets/3cccd818-0611-4e73-9861-79c6dda09dc4)
@bwklein

**Comment by drjfloyd** [comment]
This isn't possible to do with the current inputs available in FDS.
